### ISSUE TITLE
Segmentation fault: 11

### ISSUE THREAD

**Issue by wilfriedguiblet** [ViennaRNA/ViennaRNA#185]
Hi,

I am having an issue running an RNAlib function in python. Here is my script:

```
import RNA
seq = "GAGUAGUGGAACCAGGCUAUGUUUGUGACUCGCAGACUAACA" 
md = RNA.md()
fc = RNA.fold_compound(seq, md)
fc.mfe_window_zscore(1)
```

and get the error `Segmentation fault: 11`

Am I doing something wrong or is there an issue?

Note: I am on Mac and using RNAlib 2.5.0.

Thanks!

**Comment by RaumZeit** [comment]
Hi,
this is a know z-score intialization bug that will be fixed with the next release v2.6.0 of the ViennaRNA Package. For now, it should suffice if you add the following extra line before caling the `mfe_window_zscore()` method:
```
fc.zsc_filter_init()
```

Best

**Comment by wilfriedguiblet** [comment]
It works. Thanks!
